<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Driving_Dlg</class>
 <widget class="QDialog" name="Driving_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>542</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Purisa</family>
    <pointsize>15</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Driving Dialog</string>
  </property>
  <widget class="QVTKWidget" name="qvtk_velodyne_driving_dlg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>631</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="grbx_driving">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>391</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QGroupBox" name="rgbx_velodyne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Velodyne</string>
    </property>
    <widget class="QPushButton" name="bt_velodyne_connection">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbx_vehicle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Vehicle</string>
    </property>
    <widget class="QPushButton" name="bt_vehicle_run">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>44</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grbx_direction">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>27</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Direction</string>
     </property>
     <widget class="QRadioButton" name="rd_vehicle_dir_forward">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>22</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Forward</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rd_vehicle_dir_backward">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>46</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Backward</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rd_vehicle_dir_left">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>72</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rd_vehicle_dir_right">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>96</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="ed_vehicle_velocity">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>109</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_vehicle_set_option">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>102</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_vehicle_connection">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>151</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ed_vehicle_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>/dev/ttyACM0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>153</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vehicle Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rgbx_gps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>GPS</string>
    </property>
    <widget class="QPushButton" name="bt_gps_connection">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbx_device">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QCheckBox" name="ck_device_vehicle">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>28</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vehicle</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_device_velodyne">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Velodyne</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_device_gps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>72</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GPS</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_mission_driving">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Driving</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_mission_parking">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Parking</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
